### ISSUE TITLE
Weird missing user icon

### ISSUE THREAD

**Issue by local-minimum** [JammerCore/JammerCore#1892]
I commenter on my game has a missing user icon:
![bild](https://user-images.githubusercontent.com/8041100/80301334-da4b2c00-87a3-11ea-86e3-b64d89b7ee73.png)
And that looks quite bad in the feed. Should it not revert back to the default icon instead? It seems persistent and specific so I don't think it has anything to do with cache and failed loads.

**Comment by winniehell** [comment]
In case it helps debugging, the same happens for my account:

![Bildschirmfoto vom 2020-09-26 16-56-35](https://user-images.githubusercontent.com/1652293/94343570-46fb7880-0019-11eb-9a16-0647cba51234.png)

(https://ldjam.com/events/ludum-dare/46/genamoeba/ldjam-46-jingle)

The corresponding image tag does not seem to have a `src` attribute:

```html
<img alt="winniehell's avatar image">
```

On the top right I see this:

![Bildschirmfoto vom 2020-09-26 16-57-43](https://user-images.githubusercontent.com/1652293/94343592-6e524580-0019-11eb-8b0a-f28655933d0e.png)

That image is https://static.jam.vg///content/dc4/2/z/9113.jpg.40x40.fit.png which returns an error 500 and a JSON response:

```json
{
  "status": 500,
  "message": "PROC ERROR [99]: ",
  "data": {
    "user": 9421,
    "asset": {
      "width": 500,
      "height": 500,
      "mime": "image/jpeg",
      "size": 154399
    }
  }
}
```

On my profile I see the default icon (https://static.jam.vg/content/internal/user64.png):

![Bildschirmfoto vom 2020-09-26 17-00-24](https://user-images.githubusercontent.com/1652293/94343638-c8eba180-0019-11eb-91fe-cd8f61e0b17b.png)

**Comment by winniehell** [comment]
I set my avatar on 2017-08-03 and did not change it since then in case that helps narrowing done the error.

**Comment by winniehell** [comment]
> That image is https://static.jam.vg///content/dc4/2/z/9113.jpg.40x40.fit.png which returns an error 500 and a JSON response:

It looks like https://static.jam.vg///content/dc4/2/z/9113.jpg works. :thinking:

**Comment by janslifka** [comment]
> > That image is https://static.jam.vg///content/dc4/2/z/9113.jpg.40x40.fit.png which returns an error 500 and a JSON response:
> 
> It looks like https://static.jam.vg///content/dc4/2/z/9113.jpg works. 🤔

Also changing `png` to `jpg` works https://static.jam.vg///content/dc4/2/z/9113.jpg.40x40.fit.jpg

**Comment by zwrawr** [comment]
#1410

**Comment by zwrawr** [comment]
I wrote a quick script to check to see which accounts have broken 64x64 avatars. https://gist.github.com/zwrawr/b2f11dbb48e8a039f63a075b0a427707#file-brokenimg-py

Here is a list of all (maybe?) accounts with the issue. There is 250 of them.
https://gist.github.com/zwrawr/71ace267afa341a377499e9d6fcd237e

Adding `.$cmd.` to this message would report what is failing.
https://github.com/ludumdare/ludumdare/blob/a0e98b4f28f3064730253e637f08d9e0c678c730/public-static/content.php#L158

### A secondary bug seems to be going on with the extensions
* ` 163437, old-guy, https://static.jam.vg/content/d6e/72/z/25419.image/x-icon, https://static.jam.vg/content/d6e/72/z/25419.image/x-icon.64x64.fit.png `
this seems to have the extension `.image/x-icon` which is a mime type.

* `10875, felipe-nanni,   https://static.jam.vg/content/b7a/2/z/4f05.image/bmp, https://static.jam.vg/content/b7a/2/z/4f05.image/bmp.64x64.fit.png
`
This has `.image/bmp` 

* `73837, trypants, https://static.jam.vg/content/d60/21/z/f423.application/octet-stream, https://static.jam.vg/content/d60/21/z/f423.application/octet-stream.64x64.fit.png`
This has `.application/octet-stream`

**Comment by aidswidjaja** [comment]
Yep, I have this bug as well and it looks like my account is part of the 250 which have broken profile pictures.

I will try changing from a png to jpg. 

https://github.com/ludumdare/ludumdare/issues/1957

**Comment by mikekasprzak** [comment]
Resolved by #1988.
